### PR TITLE
Add insights variable 

### PR DESCRIPTION
**Pull request by arjundashrath**

### CODE REVIEW

**Comment by arjundashrath** [comment]
@varunsh-coder I wanted to enquire about a solution to the counter metrics situation. I did not tamper with the `SecureWorkflowReponse` structure as it was referenced in a lot of places. However, what I tried doing is that I tried changing the `SecureWorkflow` function by adding another return parameter of `bool` type. The idea is that whenever a workflow is fixed, a `true` value is returned by the function along with the previous parameters, else the returned value is `false`. I also added another `bool` type return parameter in the `Invoke` function in the `main.go` file and managed the return values of the `SecureWorkflow` function referenced inside the invoke function. So now essentially the Invoke function would return three values instead of two. The last value will be a bool variable that is true if the tool secured the workflow and false if it did not secure the workflow. Now, this last parameter can be used to increment a counter in the private code hosted on AWS. Wanted to know your thoughts on it if it would work? I have not managed the test cases for the changes yet.

**Comment by varunsh-coder** [comment]
> @varunsh-coder I wanted to enquire about a solution to the counter metrics situation. I did not tamper with the `SecureWorkflowReponse` structure as it was referenced in a lot of places. However, what I tried doing is that I tried changing the `SecureWorkflow` function by adding another return parameter of `bool` type. The idea is that whenever a workflow is fixed, a `true` value is returned by the function along with the previous parameters, else the returned value is `false`. I also added another `bool` type return parameter in the `Invoke` function in the `main.go` file and managed the return values of the `SecureWorkflow` function referenced inside the invoke function. So now essentially the Invoke function would return three values instead of two. The last value will be a bool variable that is true if the tool secured the workflow and false if it did not secure the workflow. Now, this last parameter can be used to increment a counter in the private code hosted on AWS. Wanted to know your thoughts on it if it would work? I have not managed the test cases for the changes yet.

Let me think about this some more. Just listing requirements:

- [ ] count how many workflows were secured
- [ ] how many unique workflows were secured
- [ ] how many could not be secured
- [ ] reason why they could not be secured
- [ ] show number secured on the website

File: secureworkflow.go [hunk 1]
```diff
@@ -10,10 +10,11 @@ const (
 	HardenRunnerActionName        = "Harden Runner"
 )
 
-func SecureWorkflow(queryStringParams map[string]string, inputYaml string, svc dynamodbiface.DynamoDBAPI) (*SecureWorkflowReponse, error) {
+func SecureWorkflow(queryStringParams map[string]string, inputYaml string, svc dynamodbiface.DynamoDBAPI) (*SecureWorkflowReponse, error, bool) {
```
Comment (K230695) [inline secureworkflow.go:13]: secureworkflow.go

**Review by K230695 — APPROVED** [review]

File: secureworkflow.go (hunk 1 above)
Comment (K230695) [reply secureworkflow.go:13]: Go

**Pull request closed without merging**